sys/include/net/loramac.h

```diff
@@ -30,7 +30,8 @@ extern "C" {
 #endif
 
 /**
- * @name    LoRaMAC default parameters
+ * @defgroup net_loramac_conf    LoRaMAC default parameters
```

Comment:
Unifying with the other configuration groups

```suggestion
 * @defgroup net_loramac_conf     LoRaMAC compile configurations
```

?

Reply:
roger